src/common/state-button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import ActionButton from './action-button'
+import Colors from './Themes/default.js'
+import propTypes from './prop-types'
+
+const Button = styled(ActionButton)`
+  background-color: #fff
+  border: 2px solid ${props => props.enabled ? Colors.enabled : Colors.disabled}
+  color: ${props => props.enabled ? Colors.enabled : Colors.disabled}
+  font-size: 12px
+  font-weight: bold
+`
+
+const StateButton = ({value, state, ...props}) =>
+  <Button
+    {...props}
+    enabled={value}
```

Comment:
`value` is true when the job is enable. This prop used to define the text color.